docs/site-components-deployment.md

```diff
@@ -80,10 +80,18 @@ CLIENT_URL=https://tf2pickup.fi
 BOT_NAME=tf2pickup.fi
 
 # MongoDB
-MONGODB_URI=mongodb://tf2pickup:yoursuperfunnypassword@mongodb:27017/tf2pickup
+# The commented values below were used in the old version of the tf2pickup.org project, deprecated since server 8.x version
+# MONGODB_HOST=tf2pickup-fi-mongo
+# MONGODB_PORT=27017
+# MONGODB_DB=admin
 MONGODB_USERNAME=tf2pickup
 MONGODB_PASSWORD=yoursuperfunnypassword
-MONGODB_DB=tf2pickup
+# MONGODB_URI syntax:
+# mongodb://username:password@hostname/database-name
+MONGODB_URI=mongodb://tf2pickup:yoursuperfunnypassword@tf2pickup-fi-mongo/admin
```

Comment:
@garrappachc Does this need to be URL encoded? Or does your handy package handle this for us?

Reply:
Yeah, it doesn't need to be: https://github.com/tf2pickup-org/server/blob/f62077e061d0d44ab4d1f63dd02cd9cf7c2b19cf/src/app.module.ts#L40

Although I'd personally test it :smiley: